app/services/api/error_factory.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Api
+  class ErrorFactory
+    attr_reader :error
+
+    def initialize(error:)
+      @error = error
+    end
+
+    def call
+      error.errors.map do |e|
+        {
+          title: "#{e.attribute.to_s.humanize} is #{e.type.to_s.humanize.downcase}",
+          detail: e.message,
+        }
+      end
+    end
```

Comment:
This is a first attempt which kind of works. You've commented on an older version. It now takes the model instead of errors as is needed if we want to do any `i18n` calls. Improvements are welcomed